helm/default-apps-aws/values.yaml

```diff
@@ -202,6 +202,29 @@ apps:
     # used by renovate
     # repo: giantswarm/observability-bundle
     version: 0.7.2
+  teleport-kube-agent:
+    appName: teleport-kube-agent
+    chartName: teleport-kube-agent-app
+    catalog: default
+    clusterValues:
+      configMap: true
+      secret: true
+    enabled: true
+    forceUpgrade: true
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/teleport-kube-agent-app
+    version: 0.3.0
+    # a list of extraConfigs for the App,
+    # It can be secret or configmap
+    # https://github.com/giantswarm/rfc/tree/main/multi-layer-app-config#example
+    extraConfigs:
+      - kind: secret
+        name: "{{ $.Values.clusterName }}-teleport-join-token"
+        namespace: "{{ $.Release.Namespace }}"
+      - kind: configMap
+        name: "{{ $.Values.clusterName }}-teleport-kube-agent-config"
+        namespace: "{{ $.Release.Namespace }}"
```

Comment:
as this will block the app deployment until the resource is avaiable

Reply:
Shouldn't be a problem cause teleport-operator will be creating those as soon as Cluster CR is reconciled, before default apps is deployed.